tests/testmatrix.c

```diff
@@ -679,6 +679,7 @@ static void test_isMatrixIdentity ()
 	assert (isIdentity == TRUE);
 
 	// Less close to identity.
+	// FIXME: it appears that GDI+ is less precise than GDI+.
```

Comment:
@mellinoe @akoeplinger do you know the algorithm GDI+ uses for `GdipIsMatrixIdentity`? I've tried narrowing the values down to around `+-0.99949f` but I don't want to start mucking around with things that potentially break

Reply:
Sorry, I don't know -- I'm really not familiar with the GDI+ codebase.

Reply:
No problem, just checking!